frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -107,8 +107,7 @@ function BookInfo:show(file, book_props)
             -- for all others than crengine, we seem to get an accurate nb of pages
             pages = document:getPageCount()
         end
-        -- via pcall because picdocument:getProps() always fails (we could
-        -- check document.is_pic, but this way, we'll catch any other error)
+        -- via pcall, even if getProps() shouldn't fail anymore
```

Comment:
Those last two sentences were more of a fictional future reader. ;-) But you seem to be the one who added the pcall? If this commit fixes the reason for it then I suppose we can just get rid of it.